settings/site.ini

```diff
@@ -562,10 +562,12 @@ UserCreatorID=14
 # md5_user generates password hash from user and password.
 # md5_site generates password hash from site, user and password
 # plaintext does not generate a hash but has the password as it is, this is not
-# recommended since it is a huge security risc.
+# recommended since it is a huge security risk.
+# bcrypt generates hash from password only
+# php_default uses whichever method PHP deems appropriate (currently bcrypt, as of PHP 7)
```

Comment:
`as of PHP 7` => `as of PHP 5.5`: http://php.net/password_hash

Reply:
Fixed, thanks.